### ISSUE TITLE
Future tasks in the future

### ISSUE THREAD

**Issue by dnxjay** [ubsuny/CP1-24-HW1#99]
I would like to entertain an open discussion about this task. I felt confused most of it due to the lack of keeping up with the tasks and who's doing what. Going forward my suggestion would be creating a better structure with doing so, do anyone else agree if so, what recommendations can we make?

**Comment by p-pxmpo** [comment]
I agree we have no reliable way of keeping track of who's doing what, everybody just seems to be doing their own thing. Maybe when a new HW comes out we can implement some sort of spreadsheet in the beginning and everyone can just type their names in and what tasks they are doing/assigned to?

**Comment by laserlab** [comment]
Good ideas. I will present another one today in class.

**Comment by kylemasc917** [comment]
A suggestion is that the first issues posted and left open should be general categories of work where people can volunteer under certain tasks and then the file maintainer should keep a list in some file in the main repository that keeps track of everyone and their tasks as well as any open tasks in case someone doesn't volunteer right away.

**Comment by kylemasc917** [comment]
This way before any actual coding begins everyone can be at least cursory aware of their tasks and who else is working on similar.

**Comment by dnxjay** [comment]
i agree felt like things were disorganized in a sense and if you work at a different pace than someone else you were left empty handed at the beginning
